pyvo/mivot/utils/xml_utils.py

```diff
@@ -0,0 +1,80 @@
+"""
+Created on 16 Dec 2021
+
+@author: laurentmichel
+"""
+from lxml import etree
```

Comment:
This is only the a test file, so should not matter that much, but I recall the question of whether use lxml or defusedxml was raised in connection with this mivot work.

Also, either of the new dependencies is being used, they need to be added properly as a dependency (whether they will be mandatory or optional one is open for discussion, but at least they have to be added as a test dependency)

Reply:
Unfortunately, lxml is used by the model_viewer module which is unavoidable. So whatever parser we will be using, we will have to add a new dependency. See [slack thread](https://astropy.slack.com/archives/CJF2R8K50/p1695050432958479).

There is here a long [discussion](https://discuss.python.org/t/status-of-defusedxml-and-recommendation-in-docs/34762) about lxml vs defused xml.